resilience4j-rxjava2/src/main/java/io/github/resilience4j/bulkhead/operator/BulkheadOperator.java

```diff
@@ -33,6 +33,7 @@
 
 /**
  * A RxJava operator which wraps a reactive type in a bulkhead.
+ * Please note that Bulkhead shouldn't use wait time as it may block while acquiring the permit.
```

Comment:
* Please note that Bulkhead shouldn't use wait time as it may block while acquiring the permit. 

can u please explain more what you mean by that ?

Reply:
If `io.github.resilience4j.bulkhead.BulkheadConfig#maxWaitTime` is set on `Bulkhead` then `io.github.resilience4j.bulkhead.internal.SemaphoreBulkhead#tryEnterBulkhead` might block the current executing thread for that long which can be problematic if that's one of the computational scheduler's threads. Would be nice to have a non-blocking way of doing this.

Reply:
Currently that's only possible by using maxWaitTime = 0

Reply:
Yes, I meant that. Should I rephrase this comment?

Reply:
Yes please